### ISSUE TITLE
Logout function

### ISSUE THREAD

**Issue by qy21gafy** [amosproj/amos2022ws05-shared-desk-mgmt#28]
## User story
1. As a user
2. I want / need to be able to logout from shared desk management
3. So that outsiders can't access the system, e.g if I logged in on a public computer

## Acceptance criteria
* After logging out the user should not be able to access any internal functions of shared desk management anymore
* The user should be able to login again after logging out

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by qy21gafy** [comment]
This is already implemented using next-auth in our frontend code. So it will be ready after the Homepage UI task
The frontend part is done.
There is no backend part for that, because we simply invalidate the session in the frontend and that causes the user to be signed out.
The work for this is done in the Basic Homepage UI task and can also be found in the branch
https://github.com/amosproj/amos2022ws05-shared-desk-mgmt/tree/homepage-ui-and-auth
